build.sbt

```diff
@@ -54,26 +54,25 @@ lazy val zioJson = crossProject(JSPlatform, JVMPlatform)
     // as per @fommil, optimization slows things down.
     scalacOptions -= "-opt:l:inline",
     scalacOptions -= "-opt-inline-from:zio.internal.**",
-
     libraryDependencies ++= Seq(
-      "com.propensive"                        %% "magnolia"              % "0.16.0",
-      "org.scalaz"                            %% "scalaz-core"           % "7.3.2" intransitive (),
-      "eu.timepit"                            %% "refined"               % "0.9.15" intransitive (),
-      "org.scala-lang"                        % "scala-reflect"          % scalaVersion.value % Provided,
-      "dev.zio"                               %% "zio"                   % zioVersion,
-      "dev.zio"                               %% "zio-streams"           % zioVersion,
+      "com.propensive"                        %% "magnolia"                % "0.16.0",
```

Comment:
Wish all version numbers where assigned to a `val` like `zioVersion` to be more consistent

Reply:
For all dependencies, or when multiple dependencies share a common value? (circe-core, circe-generic, …)